workers/r_dependencies.R

```diff
@@ -0,0 +1,205 @@
+install.packages('ff', repos='https://cloud.r-project.org/')
+install.packages('XML', repos='https://cloud.r-project.org/')
+install.packages('RCurl', repos='https://cloud.r-project.org/')
+install.packages('RSQLite', repos='https://cloud.r-project.org/')
+
+source('https://bioconductor.org/biocLite.R')
+biocLite('oligo')
```

Comment:
Is there a reason for the decision to not include the result of these packages in one vector (perhaps called `bioc.packages`) and then lapply over the whole thing?
```R
lapply(bioc.packages, biocLite)
```